### DISCUSSION TITLE
'Transfer Incomplete' error, but with different conditions than similar, past posts

### DISCUSSION THREAD

**Discussion by stevenjstromer** [iterate-ch/cyberduck#15637] [Q&A]
After years of smooth sailing, I am receiving 'Transfer Incomplete' errors when SFTPing files from my local machine to remote server. The files **do** transfer successfully. However, the transfer log window reports the error, and I note that the reported amount of uploaded data is **exactly** twice that of the actual file size, as if Cyberduck is trying to upload the file twice. This is readily apparent in the attached screenshot.

I am running Cyberduck Version 8.7.2 (40975) on OSX Ventura 13.6.3. Thanks for any wisdom on this.

<img width="658" alt="Screenshot 2024-02-14 at 11 51 01 AM" src="https://github.com/iterate-ch/cyberduck/assets/59570421/5f26a82d-ba0a-45a0-aef0-1e5b81a38b6f">

UPDATE: Adding to this that I just downgraded to version Cyberduck-8.7.1.40770, and the problem instantly disappears. I believe this to be a bug in 8.7.2. If there is anything I can do to assist the developer in log tracing or similar, please let me know. Thanks!

**Comment by tatze96** [comment]
I can't reproduce the described behavior while uploading similarly sized items to the SFTP servers available to me.

Does this behavior only occur while transferring files or also while transferring folders?
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15637 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
